assets/src/block-validation/helpers/index.js

```diff
@@ -231,7 +231,10 @@ export const maybeDisplayNotice = () => {
 		options.actions = [
 			{
 				label: __( 'Review issues', 'amp' ),
-				url: reviewLink,
+				className: 'components-button components-notice__action is-link',
```

Comment:
The classes `components-button` and `components-notice__action` are already applied to the button, so all that would be needed is `is-link`.

```suggestion
				className: 'is-link',
```

Reply:
Before:

![image](https://user-images.githubusercontent.com/134745/92063365-d762d800-ed4f-11ea-9be2-4551016d2784.png)

After:

![image](https://user-images.githubusercontent.com/134745/92063398-e5b0f400-ed4f-11ea-96fd-e97339d799a4.png)

And it still renders as a link of course:

![image](https://user-images.githubusercontent.com/134745/92063471-05e0b300-ed50-11ea-91a1-28d581139faf.png)